src/stdatamodels/jwst/datamodels/schemas/core.schema.yaml

```diff
@@ -2402,6 +2402,11 @@ properties:
             type: string
             fits_keyword: S_PHOTOM
             blend_table: True
+          pixel_replacement:
```

Comment:
Ack. I see from https://github.com/spacetelescope/jwst/pull/7398/files that the actual name of the step is in fact "pixel_replace" (not replace*ment*), so I take back my previous suggestion and this should be changed back to just "pixel_replace". Sorry about the hassle.

Reply:
No worries, flipped it back.